tools/mcsema_disass/ida/get_cfg.py

```diff
@@ -227,10 +227,8 @@ def parse_os_defs_file(df):
         # Misidentified and external.
         flags = idc.GetFlags(ea)
         if not idc.isCode(flags) and not idaapi.is_weak_name(ea):
-          DEBUG("ERROR: External {} at {:x} from definitions file is not a function".format(
+          DEBUG("WARNING: External {} at {:x} from definitions file is not a function".format(
```

Comment:
Can you add in a comment here, something like:
`# See Issue #308`, and change the wording of the debug message to `may not be a function`.